spec/models/recommendation_spec.rb

```diff
@@ -12,4 +12,31 @@
       expect(recommendation).to_not be_valid
     end
   end
+
+  describe "finding media" do
+    it "can find a new movie" do
+      VCR.use_cassette "recommendation" do
+        movie = Recommendation.find_or_create_media("looper")
+
+        expect(movie.title).to eq("looper")
+      end
+    end
+
+    it "can find a movie that exists in the database" do
+      create(:recommendation, title: "inception")
+      movie = Recommendation.find_or_create_media("inception")
+
+      expect(movie.title).to eq("inception")
+    end
+  end
+
+  describe "filtering content" do
+    it 'filters adult content' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.